src/tests/cli-tests/src/contracts/zkasm/contract.zkasm

```diff
@@ -0,0 +1,39 @@
+;! { "cases": [ {
```

Comment:
Please remove this compiler-tester metadata.

Reply:
done